micro_sam/sam_annotator/image_series_annotator.py

```diff
@@ -63,12 +63,21 @@ def image_series_annotator(
         return_viewer: Whether to return the napari viewer to further modify it before starting the tool.
         predictor: The Segment Anything model. Passing this enables using fully custom models.
             If you pass `predictor` then `model_type` will be ignored.
+<<<<<<< HEAD
```

Comment:
Just FYI there's the remnants of a merge conflict that hasn't been totally resolved still in here 😄

Reply:
Thanks for catching this @GenevieveBuckley !